package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "tslint --project tsconfig.json",
     "test": "npm run docker && ts-node tests/prepare-docker-nextcloud-container.ts && jest --config jest.conf.js --runInBand; docker-compose down -v",
     "test-watch": "npm run docker && ts-node tests/prepare-docker-nextcloud-container.ts && jest --config jest.conf.js --runInBand --watch",
-    "fast-test-watch": "ts-node tests/prepare-docker-nextcloud-container.ts && jest --config jest.conf.js --runInBand --watch"
+    "fast-test-watch": "ts-node tests/prepare-docker-nextcloud-container.ts && jest --config jest.conf.js --runInBand --watch",
+    "fast-test-watch-nostart": "ts-node tests/prepare-docker-nextcloud-container.ts && jest --config jest.conf.js --runInBand --watch --testNamePattern=stopTestingAtStart"
```

Comment:
This does what exactly?

Reply:
Allows to starting of test mode without it kicking off with all the tests.
Unfortunately there is no CLI option that allows this.